pkg/features/kube_features.go

```diff
@@ -727,8 +728,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	CSIMigrationvSphere:            {Default: false, PreRelease: featuregate.Beta},
 	CSIMigrationvSphereComplete:    {Default: false, PreRelease: featuregate.Beta},
 	RunAsGroup:                     {Default: true, PreRelease: featuregate.Beta},
-	CSIMigrationOpenStack:          {Default: false, PreRelease: featuregate.Beta}, // Off by default (requires OpenStack Cinder CSI driver)
-	CSIMigrationOpenStackComplete:  {Default: false, PreRelease: featuregate.Alpha},
+	CSIMigrationOpenStack:          {Default: true, PreRelease: featuregate.Beta},
+	CSIMigrationOpenStackComplete:  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
We are also going to need docs PR for this - https://kubernetes.io/docs/concepts/storage/volumes/#openstack-csi-migration